### ISSUE TITLE
give feedback when loading comments

### ISSUE THREAD

**Issue by kas70** [diaspora/diaspora#5611]
on desktop and mobile site

**Comment by Flaburgan** [comment]
Yeah I agree we can improve that, currently the SPV is displaying "There is no comment at the moment" and then it suddenly displays all the comment.

**Comment by goobertron** [comment]
Agree. It would be good to display 'Loading comments...', and only display 'There are no comments on this post' (better than the current phrase) only when the app has determined that there are no comments to display.

**Comment by Flaburgan** [comment]
This is particularly visible on post with **a lot** of pictures, which take a lot of time to load. See [this one](https://diaspora-fr.org/posts/2304960) for example.
